app/templates/join.hbs

```diff
@@ -1,16 +1,17 @@
 {{page-title 'Join'}}
-<section class="join__container">
-{{#if this.login.isLoading}}
-<div data-test-loading class='intro__loading'>
-    <Fa-Icon @size='2x' @icon='circle-notch' @spin={{true}} />
-</div>
-{{else}}
-{{#if this.login.userData.roles.archived}}
-<OnboardingCard>
-<h3 class="archived__heading">Archived User</h3>
-</OnboardingCard>
-{{else}}
- <Stepper />
-{{/if}}
-{{/if}}
+<section class='join__container'>
+  {{#if this.login.isLoading}}
+    <div data-test-loading class='intro__loading'>
+      <Fa-Icon @size='2x' @icon='circle-notch' @spin={{true}} />
+    </div>
+  {{else}}
+    {{#if this.login.userData.roles.archived}}
```

Comment:
what if for some reasons the API does not returns roles?
I think we should be check `null` checks

Reply:
This is @vinit717 code and I talked with him to address this comment
he said will will addressed in his next pr